gpytorch/kernels/__init__.py

```diff
@@ -9,7 +9,7 @@
 from .grid_kernel import GridKernel
 from .grid_interpolation_kernel import GridInterpolationKernel
 from .index_kernel import IndexKernel
-from .inducing_point_kernel import InducingPointKernel
+from .inducing_point_kernel import InducingPointKernel, InducingPointKernelAddedLossTerm
```

Comment:
I'm confused - is this defined in both `.mlls` and in `.inducing_point_kernel`?

Reply:
Oops! I'd originally defined it in the kernel, but it's not there anymore and I forgot to remove the import